Source/Core/VideoBackends/D3D/D3DState.cpp

```diff
@@ -207,25 +211,25 @@ u32 StateManager::UnsetTexture(ID3D11ShaderResourceView* srv)
 {
 	u32 mask = 0;
 
-	for (u32 index = 0; index < 8; ++index)
+	for (u32 index = 0; index < 16; ++index)
 	{
 		if (m_current.textures[index] == srv)
 		{
 			SetTexture(index, nullptr);
-			mask |= 1 << index;
+			mask |= (u64)1 << index;
```

Comment:
You can change all of these casts to using UINT64_C (don't use C casts for things like this; or better yet, at all)